### PR TITLE
pages.zh/osx: add Chinese translation

### PR DESCRIPTION
**Pull request by wizarot**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by agnivade** [comment]
@wizarot - Thank you for the contribution !

Kindly rename your commit msgs in the format of "<command>: add Chinese translation". And also please remove the merge commit from between so that we can rebase all the commits to master.

Feel free to ask @mebeim for any sort of git help :stuck_out_tongue_closed_eyes:

**Comment by wizarot** [comment]
> @wizarot - Thank you for the contribution !
> 
> Kindly rename your commit msgs in the format of ": add Chinese translation". And also please remove the merge commit from between so that we can rebase all the commits to master.
> 
> Feel free to ask @mebeim for any sort of git help 😝

@agnivade 
I revised it. Is that OK?

**Comment by agnivade** [comment]
It's not actually. Sorry if I was not able to explain. I want each page translation to have its own commit. And there should not be any merge commit in the branch.

What you have currently is also "okay", but we prefer to have a separate commit for each page to keep things clean.

File: pages.zh/linux/apache2ctl.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# apache2ctl
```
Comment (sbrl) [inline pages.zh/linux/apache2ctl.md]: Hrm. Did you mean to translate this page? It looks like English to me.

Comment (wizarot) [reply pages.zh/linux/apache2ctl.md]: Remove it !

**Comment by wizarot** [comment]
Hi! I changed each page translation to its own commit.
@agnivade

**Comment by HugueLiu** [comment]
The contents in {{}} should also be translated. @wizarot

**Comment by mebeim** [comment]
@HugueLiu oh you're right, we totally missed that.

**Comment by urbalazs** [comment]
Hello @wizarot and others,
IMHO creation of translated pages manually is waste of time now. I have a pull request with a proposal for translation workflow. I suggest to move your translations to a `.po` file in `i18n` folder (not exists yet) called `i18n/tldr.zh.po` and let generate translated pages with my script located in https://github.com/tldr-pages/tldr/pull/2793. See this pull request for more details. You can find a `.pot` file in the mentioned PR.

With this workflow:

- translated pages can be easy maintainable
- translators will be notified about string changes in the source
- can be generate more output (PDF, EPUB, etc.) if needed

**Comment by agnivade** [comment]
Yes, it may require extra work to move translations to a `.po` file. But since the author already has put effort in submitting a PR, I think that should be respected. We can always move to the po4a workflow when it's ready, but until then, I don't see a reason to block all translations.

**Comment by urbalazs** [comment]
I can help to move existing translations to .po files, if the final decision will be made to use .po for translations. There are some command line tool for this and I have experiences about its usage.

**Comment by agnivade** [comment]
That's great @urbalazs ! Thanks for helping us out with this. The final decision is indeed to use .po files. But since this is a new workflow, one of us need to own and understand it. @mebeim has graciously offered to lead this, but he needs some time. Let us continue with the existing workflow until then.

**Comment by wizarot** [comment]
I translated the contents in {{}}  . @HugueLiu

**Pull request merged**